docs/releasing.md

```diff
@@ -0,0 +1,21 @@
+### Releasing Federation v2
+
+Creating a federation v2 release involves the following steps:
+
+ - Localling create an annotated tag in the format v[0-9]+.[0-9]+.[0-9]+
+   - `git tag -a <tag> -m "Creating release tag <tag>"`
+   - An annotated tag is required to trigger the image build and to
+     ensure that the `kubefed2` binary is versioned correctly.
+   - At the time of writing it is not possible to create an annotated
+     tag through the github web interface.
+ - Push the tag to master
+   - `git push origin <tag>`
+   - This requires write access to the repo.
+ - Build `kubefed2` for release
+   - `make kubefed2`
+   - `cd bin`
```

Comment:
How about adding a command as follows to confirm the `kubefed2` version?

```shell
kubefed2 version
```

Reply:
both `kubefed2` and `controller-manager` have a version command option.